docs/setup.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: "databricks"
+page_title: "Provider: Databricks"
+sidebar_current: "docs-databricks-setup"
+description: |-
+  Databricks sync databricks.
+---
+
+# Databricks-Sync
+
+Databricks-Sync is for workspace disaster recovery. You can use it to import or export some, or all, objects within a workspace. Databricks-Sync can also be used as a migration utility.
+
+## Dependencies
+
+This project requires the following environment dependencies:
+
+* [golang 1.13.x](https://golang.org/dl/)
+* [terraform 0.12.x](https://www.terraform.io/downloads.html)
+* [terraform-provider-databricks 0.2.x](https://registry.terraform.io/providers/databrickslabs/databricks/latest)
+
+## Installation
+
+0. Download: `$ git clone https://github.com/databrickslabs/terraform-provider-databricks.git`
+1. Install Databricks Terraform provider: `$ curl https://raw.githubusercontent.com/databrickslabs/databricks-terraform/master/godownloader-databricks-provider.sh | bash -s -- -b $HOME/.terraform.d/plugins`
+2. Ensure Terraform and Golang dependencies (install via the package manager of your choice):
+    1. Verify `terraform` version 0.12.x: `$ terraform version`
+    2. Verify `golang` version 1.13.x: `$ go version`
+    3. Verify `make` version 3.81: `$ make --version`
+3. Install other dependencies: `$ cd terraform-provider-databricks && pip install -r requirements.txt`
+4. Install this project: `$ make shared install`
+5. Test that the installation succeeded: `databricks-terraformer --version`
+
+## Authentication
+
+### [Databricks-CLI](https://docs.databricks.com/dev-tools/cli/index.html)
```

Comment:
Can we move the CLI section to a Pre-requisite?
I think it'll make more sense and some can skip it if it's already setup for them

Reply:
I moved this also toe prerequisites.md